product-service/services/products.service.js

```diff
@@ -0,0 +1,17 @@
+import products from '../products.json';
+
+export default class ProductsService {
+    constructor( products ){
```

Comment:
I think this constructor could be removed. Anyway you use that initialized field)